frontend/src/pages/LocationSelection/LocationSelection.tsx

```diff
@@ -4,13 +4,11 @@ import './LocationSelection.css';
 
 const LocationSelection: React.FC = () => {
   // Sample locations list
-  const locations = ["Cassie Campbell", "Option 1", "Option 2"];
+  const locations = ["Cassie Campbell"];
 
   // State to store the selected location
   // Use this state to access the user selected location for queue purposes
-  // By default, select the first location (Cassie Campbell)
-  const [selectedLocation, setSelectedLocation] = useState<string>(locations[0]);
-
+  const [selectedLocation, setSelectedLocation] = useState<string>('');
```

Comment:
Just want to confirm that it's expected behaviour to not have the default location as Cassie Campbell